spec.yaml

```diff
@@ -89,10 +89,16 @@ paths:
                 "code": "RESOURCE_CREATED",
                 "message": "A new cluster has been created with ID 'wqtlq'"
               }
-        default:
-          description: Error
+        "401":
+          description: Permission Denied
```

Comment:
It ends up looking like this when I use 'default':
<img width="450" alt="screen shot 2017-04-20 at 13 13 09" src="https://cloud.githubusercontent.com/assets/455309/25228066/357f15ca-25cb-11e7-953f-6e78bec6fe0b.png">